Kontent.Ai.Delivery/DeliveryClient.cs

```diff
@@ -363,6 +364,48 @@ public async Task<IDeliverySyncResponse> GetSyncAsync(string continuationToken)
             return new DeliverySyncResponse(response, itemModels);
         }
 
+        /// <summary>
+        /// Returns a feed that is used to traverse through strongly typed parent content items matching the optional filtering parameters.
+        /// </summary>
+        /// <param name="codename">The codename of a content item.</param>
+        /// <param name="parameters">A collection of query parameters for filtering.</param>
+        /// <returns>The <see cref="DeliveryUsedInItems"/> instance that can be used to enumerate through content item parents for the specified item codename. If no query parameters are specified, default language parents are enumerated.</returns>
+        public IDeliveryItemsFeed<IUsedInItem> GetItemUsedIn(string codename, IEnumerable<IQueryParameter> parameters = null)
```

Comment:
existing methods accepting a `codename` parameter have null handling in place. consider adding it to all usedIn methods as well:

```csharp
if (codename == null)
{
    throw new ArgumentNullException(nameof(codename), "The codename of a content item is not specified.");
}
```